awx/api/serializers.py

```diff
@@ -5376,10 +5387,11 @@ class InstanceSerializer(BaseSerializer):
     jobs_running = serializers.IntegerField(help_text=_('Count of jobs in the running or waiting state that are targeted for this instance'), read_only=True)
     jobs_total = serializers.IntegerField(help_text=_('Count of all jobs that target this instance'), read_only=True)
     health_check_pending = serializers.SerializerMethodField()
+    peers = PeersSerializer(many=True, required=False)
 
     class Meta:
         model = Instance
-        read_only_fields = ('ip_address', 'uuid', 'version')
```

Comment:
We're allowing `ip_address` to be changed?  Doesn't that have the same problem as changing the `port` value?  A bunch of instances pointing to it will have to have their configs updated.